_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,266 @@
+
+## Assumptions
+
+This rule assumes that the apparent cause of a [form field error indicator][] is the actual cause for the [form field error indicator][] being shown. If the actual cause is different, then this rule may generate incorrect results.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 3.3.1: Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification)
+- [G84: Providing a text description when the user provides information that is not in the list of allowed values](https://www.w3.org/WAI/WCAG21/Techniques/general/G84)
+- [G85: Providing a text description when user input falls outside the required format or values](https://www.w3.org/WAI/WCAG21/Techniques/general/G85)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error.
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		console.log(isNaN(age))
+		if (!age || isNaN(age)) {
+			document.getElementById('error').innerText = 'Age must be a number'
+		} else if (age < 1) {
+			document.getElementById('error').innerText = 'Age must be at least 1'
+		}
+	}
+</script>
+
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error"></span><br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+#### Passed Example 2
+
+The multiple `input` elements share a [form field error indicator][] that identifies the elements unfilled (by referencing their labels) and describes the cause of the error.
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		if (document.getElementById('name').value.length === 0) {
+			document.getElementById('error').innerText += 'You must fill the name field. '
+		}
+		var color = document.forms[0].color.value
+		if (color.length === 0) {
+			document.getElementById('error').innerText += 'You must pick a color.'
+		}
+	}
+</script>
+
+<form>
+	<h2 id="error"></h2>
+	<label for="name">Name (required)</label>
+	<input type="text" id="name" required />
+	<br />
+	<label for="address">Address</label>
+	<input type="text" id="address" />
+	<p>Pick a color (required)</p>
+	<label><input type="radio" name="color" value="blue" required />Blue</label>
+	<label><input type="radio" name="color" value="yellow" />Yellow</label>
+	<br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The `input` element does not have a [form field error indicator][].
```

Comment:
Neither does it show an error indicator. This is incorrect.

Reply:
You are correct. This is in fact a passed example!

Reply:
@WilcoFiers What exactly do you mean with "this is incorrect". Are we not allowed to use this as a passed example? This is applicable right? And it should pass this rule right? So it is a good passed example right?